src/components/InlineWrapper/InlineWrapper.docs.mdx

```diff
@@ -0,0 +1,23 @@
+import { Story, Preview } from '@storybook/addon-docs/blocks';
+import InlineWrapper from './InlineWrapper';
+
+# InlineWrapper
```

Comment:
This name feels a bit vague and doesn't really tell me when it's appropriate to use this. Could we keep the Callout name? Shopify Polaris call theirs a CalloutCard which I kind of like. https://polaris.shopify.com/components/structure/callout-card

How about CalloutWrapper?

Reply:
I agree - I like callout card a lot. Wrapper is so vague - we wrap everything 😂